wpilibc/sim/src/Utility.cpp

```diff
@@ -7,14 +7,16 @@
 
 #include "Utility.h"
 
-#include <cstdio>
-#include <iostream>
-#include <sstream>
 #if not defined(_WIN32)
```

Comment:
We didn't notice this earlier, but this is using the keyword `not`, which we generally don't use.  Change to `#ifndef _WIN32` or `#if !defined(_WIN32)`?